client/platform/web-dev/settings.js

```diff
@@ -3,6 +3,16 @@ import { loadFileAjaxSync } from "../utils"
 const Settings = JSON.parse(
   loadFileAjaxSync("/api/admin/dictionary", "application/json")
 )
-const Version = "Dev-Mode"
+const VERSION_QUERY = {
+  query: "{\n  projectVersion\n}\n"
+}
+
+const Version = JSON.parse(
+  loadFileAjaxSync(
+    "/graphql",
+    "application/json",
+    JSON.stringify(VERSION_QUERY)
+  )
+).data.projectVersion
```

Comment:
Suggested change:
```
diff --git a/client/platform/web-dev/settings.js b/client/platform/web-dev/settings.js
index c18221fb8..6ce055b87 100644
--- a/client/platform/web-dev/settings.js
+++ b/client/platform/web-dev/settings.js
@@ -1,17 +1,20 @@
+import { gql } from "apollo-boost"
 import { loadFileAjaxSync } from "../utils"
 
 const Settings = JSON.parse(
   loadFileAjaxSync("/api/admin/dictionary", "application/json")
 )
-const VERSION_QUERY = {
-  query: "{\n  projectVersion\n}\n"
-}
+const GQL_GET_VERSION_INFO = gql`
+  query {
+    projectVersion
+  }
+`
 
 const Version = JSON.parse(
   loadFileAjaxSync(
     "/graphql",
     "application/json",
-    JSON.stringify(VERSION_QUERY)
+    JSON.stringify({ query: GQL_GET_VERSION_INFO })
   )
 ).data.projectVersion
```

Reply:
Removed the `gql`, otherwise the request wasn't being processed by server